src/features/user/CompleteSignup/index.tsx

```diff
@@ -199,20 +226,22 @@ export default function CompleteSignup(): ReactElement | null {
     reset();
   }, [type]);
 
-  const createButtonClicked = async (data: any) => {
+  const createButtonClicked = async (data: CreateUserRequest) => {
     if (!acceptTerms) {
```

Comment:
Over here, `data` will not be of type `CreateUserRequest`, so by using this type, you may not catch warnings regarding invalid type for `submitData`. 

Inspect `data` and create the appropriate type (`CreateUserRequest` could be adapted/extended in some manner). 

`submitData` should be of type `CreateUserRequest`, but `data` will not be (it will only contain data from the form)